Source/Core/Core/HW/VideoInterface.cpp

```diff
@@ -775,4 +777,42 @@ void Update(u64 ticks)
   UpdateInterrupts();
 }
 
+// Create a fake VI mode for a fifolog
+void FakeVIUpdate(u32 xfb_address, u32 fb_width, u32 fb_height)
+{
+  u32 fb_stride = fb_width;
+
+  bool interlaced = fb_height > 480 / 2;
+  if (interlaced)
+  {
+    fb_height = fb_height / 2;
+    fb_stride = fb_stride * 2;
+  }
+
+  m_XFBInfoTop.POFF = 1;
+  m_XFBInfoBottom.POFF = 1;
+  m_VerticalTimingRegister.ACV = fb_height;
+  m_VerticalTimingRegister.EQU = 6;
+  m_VBlankTimingOdd.PRB = 502 - fb_height * 2;
+  m_VBlankTimingOdd.PSB = 5;
+  m_VBlankTimingEven.PRB = 503 - fb_height * 2;
+  m_VBlankTimingEven.PSB = 4;
+  m_PictureConfiguration.WPL = fb_width / 16;
+  m_PictureConfiguration.STD = fb_stride / 16;
+
+  UpdateParameters();
+
+  u32 total_halflines = GetHalfLinesPerEvenField() + GetHalfLinesPerOddField();
+
+  if ((s_half_line_count - s_even_field_first_hl) % total_halflines <
+      (s_half_line_count - s_odd_field_first_hl) % total_halflines)
+  {  // Even/Bottom field is next.
```

Comment:
Small nitpick, comments shouldn't be on the same line.

Reply:
Thanks!  I believe I copied this from another piece of code.  I will fix it.